services/github-service-api/src/main/java/org/kontinuity/catapult/service/github/api/GitHubService.java

```diff
@@ -54,4 +54,13 @@ GitHubWebhook createWebhook(GitHubRepository repository,
                                 GitHubWebhookEvent... events)
             throws IllegalArgumentException;
     
+    /**
+     * Deletes all webhooks in a specific GitHub repository
+     * 
+     * @param repository - the value object that represents the GitHub repository
+     * @throws IOException
+     * @throws IllegalArgumentException If the parameter is unspecified
+     */
+    void deleteWebhooks(final GitHubRepository repository) throws IOException, IllegalArgumentException;
```

Comment:
1) I agree with you, but just to merge this PR once at all can we keep this change and then I can handle in #61?
2) I'm wondering about a complete cleanup, but then I realize that the user might already have other webhooks. I'll make a change to delete only the one created by the test.
3) Agreed. Will change that.